File: StartSkripte/Ruby/helloWorld.rb
#!/usr/bin/env ruby

# interaktive Konsole:
# rbi

# runMe: 
# ruby helloWorld.rb

class MyGreeter
    attr_accessor :name
    
    def initialize(name = "Welt")
        @name = name
    end
  
    def sag_hallo()
        puts "Hallo, #{@name}!"
    end

end


s = MyGreeter.new("Tim und Struppi");
s.sag_hallo();

MyGreeter.new().sag_hallo()

File: StartSkripte/Haskell/HelloWorld.hs
-- interaktive Konsole:
-- ghci
-- beenden mit :quit

-- runMe:
-- ghc HelloWorld.hs; ./HelloWorld

add :: Int -> Int -> Int
add x y = x + y

double :: Int -> Int
double x = add x x

main = do
    print ("Hello " ++ "World")
    print (double 5)

